spi_nand_flash/Kconfig

```diff
@@ -0,0 +1,9 @@
+menu "SPI Nand Flash driver"
+
+    config SPI_FLASH_NAND_ENABLED
```

Comment:
Is there any reason to introduce this option? If a user adds the component to the build, typically its functionality should be available without additional configuration. Or, putting it the other way around: is there any use case where we would add this new component and disable this option?

Reply:
It was added by the author for some different reason. But it is not needed anymore as we are making it as independent component. Updated